src/makeRobot.js

```diff
@@ -38,7 +38,80 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const template = {
+    x: 0,
+    y: 0,
+    name: name,
+    wheels: wheels,
+    version: version,
+    get info() {
+      const infoStr = `name: ${this.name}, `
```

Comment:
```suggestion
      return `name: ${this.name}, `
```
Можна одразу ж повертати необхідний рядок

Reply:
привіт, спершу я і пробував повертати interpolated string, але втручається **eslint(max-len)** - більше 80 символів не проходить перевірку лінтера ¯\_(ツ)_/¯
![lint error max line length](https://user-images.githubusercontent.com/39308717/133094998-7bcd2358-ff41-4af6-acef-7cc2c2b2287a.png)